YoCode/FileImport.cs

```diff
@@ -27,10 +27,20 @@ public static IEnumerable<string> GetAllFilesInDirectory(string path)
             }
         }
 
-        //Helper method to convert FileInfo[] elements to string and add them to a list 
-        public static void AddFileInfoToList(List<string> files, IEnumerable<FileInfo> fileinfo)
+        public static void AddFileInfoToList(List<string> files, IEnumerable<FileInfo> fileinfo, string path)
         {
-            files.AddRange(fileinfo.Select(fi => fi.ToString()));
+            foreach (var f in fileinfo)
+            {
+                if (FileIsNotInBuildDirectory(path, f))
+                {
+                    files.Add(f.ToString());
+                }
+            }
+        }
+
+        private static bool FileIsNotInBuildDirectory(string path, FileInfo fileinfo)
+        {
+            return !Path.GetRelativePath(path, fileinfo.DirectoryName).Contains("obj") && !Path.GetRelativePath(path, fileinfo.DirectoryName).Contains("bin");
         }
 
         public FileStream GetFileStream(string path)
```

Comment:
I noticed this GetFileStream method is not used at all and can be removed. The whole FileImport class could be made static. Not very important, just thought I'd mention it.